### PR TITLE
MARS MUSICAL FOUNTAIN

### PR DESCRIPTION
**Pull request by kshitij2116**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
-# Musical-Fountain
-Srishti 2019
+
+#MUSICAL FOUNTAIN  
+
+![Image 1](https://github.com/kshitij2116/Musical-Fountain/blob/master/Images%20and%20Videos/Images/IMG_2318.JPG)
+![Image 2](https://github.com/kshitij2116/Musical-Fountain/blob/master/Images%20and%20Videos/Images/IMG_2327.JPG)  
+
+##ABSTRACT
```
Comment (agshubhs) [inline README.md]: In all the heading there should be space after ##. That's why you are not getting the desired output.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+We also added a stepper motor which rotates the entire revolute at a constant 30 steps per second. We also added another stepper motor which would control a central valve.
+
+The nozzles were 3d printed to reduce the mannufacturing cost. For insulation, we mostly used locally available epoxy glue and araldite. 
+
+##MOTIVATION
+
+From the outside , musical fountain seemed to be a trivial project. but, when we got into its technical aspects, it wasn't so. Everytime we came up with a design, we had to brainstorm it and eventually a new design came into picture. This challenge of coming up with an efficient model is what kept us motivated.
+
+##ELECTRONIC ASPECT  
+
```
Comment (agshubhs) [inline README.md]: Write this as well as Mechanical Aspect section in points.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,60 @@
...
+##MECHANICAL ASPECT  
+
+The revolute of the model was based on The Lazy Susan armature. we 3d printed the nozzles and 2 gears with 50 and 25 teeth respectively. 6 hollow tubes were attached on the lazy susan, at the end of each was a servo motor attached which would eventually control the movement of the nozzle. A secondary base was manufactured to setup the stepper motor which was used to rotate the armature. the secodary base also housed another stepper motor which would be controlling the valve. 3 pumps were used to pump water in the fountain.  
+
+##APPLICATION  
+
```
Comment (agshubhs) [inline README.md:53]: Write this section in points.

**Comment by agshubhs** [comment]
@kshitij2116 
1. I have mention some changes in Readme.md see them.
2. There are two nested folder mechanical design/Solid works file. Remove the inner folder and paste all its files inside the Mechanical Design Folder.
3.  Add good images that are clear. The images you have inserted are not clear.
4. Add image of the final Solidworks Design in Mechanical Design folder nad them in read in mechancial aspect part.
5. Insert one image of the team members in README.md after Mentor section.

**Comment by kshitij2116** [comment]
Ok sir. Will be able to reflect the changes after 6 today. In class right
now

File: README.md [hunk 4]
```diff
@@ -1,2 +1,61 @@
-# Musical-Fountain
-Srishti 2019
```
Comment (agshubhs) [inline README.md:2]: Add this line Srishti 2019

File: README.md [hunk 5]
```diff
@@ -1,2 +1,61 @@
...
+
+# MUSICAL FOUNTAIN  
+
+![Image 1](https://github.com/kshitij2116/Musical-Fountain/blob/master/Images%20and%20Videos/Images/IMG_2313.JPG)
+![Image 2](https://github.com/kshitij2116/Musical-Fountain/blob/master/Images%20and%20Videos/Images/IMG_2327.JPG)
```
Comment (agshubhs) [inline README.md]: Remove 2nd and 3rd image. One image is sufficient in readme file

File: README.md (hunk 2 above)
Comment (agshubhs) [reply README.md]: Write this section in points

**Comment by agshubhs** [comment]
@kshitij2116  
1. Add jpeg image of the Solidworks Design in the Mechanical design Folder.
2. Heading of each section should be written normally not in capital letters.
3. Remove extra folder from the folder mechanical Design.
4. The folder name  should be **Mechanical Design** , the first letter should be capital.

**Comment by agshubhs** [comment]
@kshitij2116  
1. Readme is not giving the desired output. See it.
2. Rename all the images according to their description in Images Folder.
3. Remove folder **SolidWorks FILE** from Mechanical Design folder.

**Comment by kshitij2116** [comment]
Sir i am not able to remove the solid works folder. I cant even find it on
the net as to how to do it. And for the output part, i wrote the codes as
per your previous guidelines

**Comment by agshubhs** [comment]
use **Github desktop** software to do these things easily. you can easily delete the files update the readme, rename files easily in Windows.

**Comment by agshubhs** [comment]
@kshitij2116 
1. Solid work file folder is still not deleted.
2. Insert the cropped image of solid work design that consists of the design only.
3. First lin in README.md file.
4. Electronics Aspect and Mechanical Aspects should be in points.

**Comment by kshitij2116** [comment]
Sir bash ke through nahi samajh aa raha kaise delete karne hai. Online jo
commands hai wo nahi samajh aa rahe. And readme file ki first line waise hi
copy ki jaisa aapne kaha tha. Fir bhi output nahi aa raha. Please help

**Comment by agshubhs** [comment]
@kshitij2116 
1. Images are not rendered in the Readme file.
2. Two mechanical Design folders are present.

**Comment by agshubhs** [comment]
@kshitij2116  
Cost structure section is missing in readme

**Comment by agshubhs** [comment]
@kshitij2116 
1. Cropped the image fully so that it contains the design only.
2. Add cost structure table also

**Comment by agshubhs** [comment]
@kshitij2116 
Good Work. 
Looks good to me.

**Comment by kshitij2116** [comment]
Thanks sir

**Pull request merged**